python/openassetio/test/manager/apiComplianceSuite.py

```diff
@@ -68,3 +81,71 @@ def test_is_correct_type(self):
 
     def test_matches_fixture(self):
         self.assertEqual(self._fixtures["info"], self._manager.info())
+
+
+class Test_managementPolicy(FixtureAugmentedTestCase):
+    """
+    Check plugin's implementation of managerAPI.ManagerInterface.managementPolicy
+    """
+
+    def test_when_called_with_single_specification_returns_single_result(self):
+        context = self._session.createContext()
+        self.__assertPolicyResults(1, context)
+
+    def test_when_called_with_ten_specifications_returns_ten_results(self):
+        context = self._session.createContext()
+        self.__assertPolicyResults(10, context)
+
+    def test_calling_with_read_context(self):
+        context = self._session.createContext()
+        context.access = context.kRead
+        self.__assertPolicyResults(1, context)
+
+    def test_calling_with_write_context(self):
+        context = self._session.createContext()
+        context.access = context.kWrite
+        self.__assertPolicyResults(1, context)
+
+    def test_calling_with_read_multiple_context(self):
+        context = self._session.createContext()
+        context.access = context.kReadMultiple
```

Comment:
Multiple isn't about a query batch, its about the number of logical assets being addressed. For example, if I want to browse for a place to save _an_ image to, the manager may have a different set of allowed targets than if I wanted to save _several_ images to. e.g, in the single case, it may allow you to target an existing image entity for a "new version", but for multiple it would be required to only target some kind of container entity.  Another more straight forward example - I'd like a browser that allows multiple selection of entities vs one.

Reply:
Ahh! That makes sense, thanks.